SConstruct

```diff
@@ -100,7 +90,7 @@ from buildutils import (Option, PathOption, BoolOption, EnumOption, Configuratio
 
 # ensure that Python and SCons versions are sufficient for the build process
 EnsurePythonVersion(3, 7)
```

Comment:
```suggestion
EnsurePythonVersion(3, 10)
```
Should this be updated?

Reply:
I was a little undecided on this, since technically you don't need to use the same Python version for compilation as what the Python module is built for. However, it's kind of confusing to keep the different requirements straight, so I updated this as suggested.

Reply:
Definitely true! The 3.8 suggestion would have been a middle ground where you don't have issues with f-string while being less restrictive. The consistent 3.10 option is likely less confusing, though.